apis/gkehub/v1beta1/generate.sh

```diff
@@ -30,8 +30,6 @@ go run . generate-types \
 #   --service google.cloud.gkehub.v1beta --api-version gkehub.cnrm.cloud.google.com/v1beta1
 
 # NOTYET - not yet following full pattern
-rm ${REPO_ROOT}/apis/gkehub/v1beta1/membershipfeaturespec_identity.go
-rm ${REPO_ROOT}/apis/gkehub/v1beta1/membershipfeaturespec_reference.go
```

Comment:
@justinsb  Do you have any context about this? I saw you added the generated.sh for gkehub orignally? It looks like this service has many exceptions. can I ask why we still include them in the validate-generated-fiels presubmit check?